invenio_base/__init__.py

```diff
@@ -53,8 +53,8 @@
 information.
 
 
-Starting a new project
-~~~~~~~~~~~~~~~~~~~~~~
+Creating a new Invenio instance
```

Comment:
The creation of the instance is the first thing people do when they are still newbies, so I preferred to clarify "which instance" is created a bit more explicity. Kinda not hurting to repeat stuff in the beginnings.